docs/source/about/advanced_usage.rst

```diff
@@ -883,6 +883,22 @@ keybindings
 
       ping_timeout = 10000
 
+`outgoing_bandwidth <https://localhost:47990/config/#outgoing_bandwidth>`__
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Description**
+   Bandwidth limitation in bytes/second to manually deal with local network throttling when the host has a greater data rate than the stream.
+   Note: for unlimited bandwidth use 0
+   Example: 1Gbps = 125000000 MBps
```

Comment:
That should be 125000000 Bps, not MBps. Anyway, it's probably best to use Mbps as unit in Sunshine's config. (Like Moonlight)

Reply:
AFAICT the issue is not so much that "the host has a greater data rate than the stream." but that there is a speed mismatch somewhere in the network path which leads to a buffer overrun in e.g. the switch or the AP. (Many users seem to use quite complex network paths between their host and the client. Quite possibly fast connections on both end devices, but rather slow internet connection in-between.)

Maybe something like "Bandwidth limitation in Mbps. Throttling the outgoing traffic can be necessary if there is a speed mismatch anywhere in the network path between the host and the client. E.g. the host is connected via 2.5Gbps LAN, but the client uses a 200 Mbps WLAN connection. The bandwidth limit should ideally be set in accordance to the slowest segment on the network path. It should also be at least twice the requested stream bandwidth."

The "twice" may not even be enough. Some more headroom may be required for VBR encoding rates and such. Not sure.